### PR TITLE
Adding option to specify root directory

### PR DESCRIPTION
**Pull request by dustinsavery**
Nextron defaults to the `renderer` directory, which is fine for the majority of cases. However, when sharing code or when needing to change that directory, there is currently no option to do so. This PR remedies that.

### CODE REVIEW

**Comment by saltyshiomix** [comment]
Hi @dustinsavery !

Thank you for sending the PR 🥇 

I think your suggestion is right, but let us think about these points:

- currently nextron watches `main` and `renderer` directories, so it may confusable for someone to think that `--dir` option is the root directory.
- what about configuring in the `nextron.config.js` like `{ rendererDir: './renderer' }`?
- and finally, if the `--dir` option is best, we should implement `bin/nextron-build`, which is duplicate.

Please tell me your ideas.

Thanks in advance :)

**Comment by saltyshiomix** [comment]
I think `app` dist directory is the way of Electron, because these boilerplates use `app` directory:

- [szwacz/electron-boilerplate](https://github.com/szwacz/electron-boilerplate)
- [chentsulin/electron-react-boilerplate](https://github.com/chentsulin/electron-react-boilerplate)

For now, it covers all by README documentation 👍

**Comment by saltyshiomix** [comment]
But I found some points:

- nextron is a template generator, so it holds `main` and `renderer` directories.
  - in other words, this PR affects `bin/nextron-init` command.
- also it affects `bin/nextron-build`.
- and and I fogot it, `distDir` (`app`) is **NOT CHANGEABLE**. See https://github.com/saltyshiomix/nextron/issues/50#issuecomment-533395649
- I think `srcRenderer` is better than `srcDir`, because we may need to have `srcMain` option.

Do you like this options?

```js
// nextron.config.js
module.exports = {
  srcMain: 'main',
  srcRenderer: 'renderer',
  webpack: (config) => {},
}
```

**Comment by dustinsavery** [comment]
> in other words, this PR affects bin/nextron-init command.
I don't believe it would affect this. By default, all directories would remain the same on init. Only by overriding it in the `nextron.config.js` would it change. So I think it would still be safe as is.

> also it affects bin/nextron-build
I have updated this accordingly above.

> I think srcRenderer is better than srcDir, because we may need to have srcMain option.
Given the need for both, `srcDir` isn't clear enough. However, `srcMain` and `srcRenderer` are pretty unclear (IMO) as well. Not sure what better names would be though...

**Comment by saltyshiomix** [comment]
> I don't believe it would affect this. By default, all directories would remain the same on init. Only by overriding it in the nextron.config.js would it change. So I think it would still be safe as is.

Sorry I misunderstood, you're right 👍

File: bin/nextron-build [hunk 1]
```diff
@@ -60,8 +60,7 @@ async function build(args) {
   // Ignore missing dependencies
   process.env.ELECTRON_BUILDER_ALLOW_UNRESOLVED_DEPENDENCIES = 'true';
   const { getUserConfig } = require('./webpack/helpers');
-  const srcDir = getUserConfig().srcDir || 'renderer';
-  const distDir = getUserConfig().distDir || 'app';
+  const rendererSrcDir = getUserConfig().rendererSrc || 'renderer';
```
Comment (saltyshiomix) [inline bin/nextron-build]: Oh, I found some typos. I'll fix them later.

**Comment by saltyshiomix** [comment]
I proudly merged this PR 🥇

**Comment by saltyshiomix** [comment]
@dustinsavery 

I just published `nextron@5.12.4` including your PR 👍

**Pull request merged**